doc/docs.xml

```diff
@@ -288,17 +288,13 @@
 	<refsect1>
 		<title>See Also</title>
 		<para>
-			<ulink url="http://conky.sourceforge.net/">
+			<ulink url="http://conky.sourceforge.net">
```

Comment:
Can you change this to `https://github.com/brndnmtthws/conky` (x2)? It redirects. Skip that.